player/lua/console.lua

```diff
@@ -546,7 +547,8 @@ local function update()
 
     local lines_max = calculate_max_log_lines()
     -- Estimate how many characters fit in one line
-    local width_max = math.ceil(screenx / opts.font_size * get_font_hw_ratio())
+    local width_max = math.floor((screenx - margin - mp.get_property_native('osd-margin-x')) /
```

Comment:
Shouldn't `osd-margin-x` should get subtracted from screenx before it gets divided by `dpi_scale`?

Reply:
I've done that for the margin too.

Reply:
Actually it doesn't seem right to substract the margins from the clip coordinates, so I divided them by the scale afterwards.